monolithe/generators/lang/javascript/writers/apiversionwriter.py

```diff
@@ -1,9 +1,16 @@
 from monolithe.generators.lib import TemplateFileWriter
+from monolithe.specifications import SpecificationAttribute
 import os
 import shutil
 
-base_attrs = ['entityScope', 'externalID', 'lastUpdatedBy'];
-named_entity_attrs = ['name', 'description'];
+base_attrs = ['entityScope', 'externalID', 'lastUpdatedBy']
+named_entity_attrs = ['name', 'description']
+iptype_enum_attr = SpecificationAttribute()
+iptype_enum_attr.name = 'IPType'
+iptype_enum_attr.allowed_choices = ['IPv4', 'IPv6', 'DUALSTACK', 'IPv4Network', 'IPv6Network']
+enabled_enum_attr = SpecificationAttribute()
+enabled_enum_attr.name = 'enabled'
+enabled_enum_attr.allowed_choices = ['DISABLED', 'ENABLED', 'INHERITED']
```

Comment:
Can you please add PermittedAction enum as well? with: USE, READ, ALL, INSTANTIATE, EXTEND, DEPLOY;